src/notification/logstream.ts

```diff
@@ -0,0 +1,32 @@
+import {Print, logger} from '../logger';
+import {LinkPollEvent} from './link_poll_event';
+
+function assertNever(x: never): never {
+  throw new Error('Unexpected object: ' + x);
+}
+
+export function sendLogstream(pollEvent: LinkPollEvent) {
+  const {link, store} = pollEvent;
+  if (pollEvent.result === 'in_stock') {
+    logger.info(`${Print.inStock(link, store, true)}\n${pollEvent.url}`);
+  } else {
+    switch (pollEvent.failureReason) {
+      case 'captcha':
+        logger.warn(Print.captcha(link, store, true));
+        return;
+      case 'banned_seller':
+        logger.warn(Print.bannedSeller(link, store, true));
+        return;
+      case 'out_of_stock':
+        logger.info(Print.outOfStock(link, store, true));
+        return;
+      case 'max_price':
+        logger.info(Print.maxPrice(link, store, pollEvent.maxPrice, true));
+        return;
+      case 'request_failed':
+        return;
+      default:
+        assertNever(pollEvent);
+    }
+  }
+}
```

Comment:
All the logging w.r.t polling had been moved here. Some logging still happends in `lookup.ts` but looked ancillary, except for what's mentioned in the comments on `notifications`.